source/app/service-providers/config/get-config-template.ts

```diff
@@ -227,6 +227,19 @@ export default function getConfigTemplate (): ConfigOptions {
       zoomBehavior: 'gui' // Used to determine what gets zoomed: The GUI or the editor
     },
     checkForBeta: false, // Should the user be notified of beta releases?
+    customizeToolbar: {
```

Comment:
I would not call this property group `customizeToolbar` because while this is the *intention of this PR*, it is not the function these settings perform. Rather something like `availableToolbarButtons` or `showToolbarButtons`. Feel free to think about a good, descriptive name yourselves!

Reply:
How about `displayToolbarButtons`?